app/assets/javascripts/app/account_transaction_report.coffee

```diff
@@ -0,0 +1,26 @@
+class AccountTransactionReport
+  constructor: (@$element) ->
+    return unless @$element.length > 0
+    @registerExportAllHandler()
+
+  registerExportAllHandler: ->
+    $('#js--exportAll').click (event) => @exportAllClicked(event)
```

Comment:
Here, I believe you can just do `$('#js--exportAll').click @exportAllClicked`. The function can be passed directly in to the click handler.